blugon.1

```diff
@@ -1,7 +1,4 @@
-.\" Manpage for blugon
-.\" author: Felix Springer <felixspringer149@gmail.com>
-.\" github: https://github.com/jumper149/blugon
-.TH man 8 "26 January 2020" "1.12.0" "blugon man page"
+.TH BLUGON 1 "26 January 2020" "blugon 1.12.0" ""
```

Comment:
All that makes sense, info in comments is pretty much overly explicit.

I think the last field for the title header should be `"blugon"`:
```groff
.TH BLUGON 1 "26 January 2020" "blugon 1.12.0" "blugon"
```